wgsl/index.bs

```diff
@@ -1768,13 +1762,13 @@ Such variables are declared with [=group=] and [=binding=] decorations.
     var<workgroup> worklist: array<i32,10>;
 
     [[block]] struct Params {
-      [[offset(0)]] specular: f32;
-      [[offset(4)]] count: i32;
+      specular: f32;
+      count: i32;
     };
     var<uniform> param: Params;          // A uniform buffer
 
     [[block]] struct PositionsBuffer {
-      [[offset(0)]] pos: [[stride(8)]] array<vec2<f32>>;
+      pos: [[stride(8)]] array<vec2<f32>>;
```

Comment:
:heart:  the fact we don't have to provide both array stride and the offset any more